<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupAppEdit</class>
 <widget class="QDialog" name="StartupAppEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Startup App</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="spacing">
    <number>15</number>
   </property>
   <item row="6" column="1" alignment="Qt::AlignRight|Qt::AlignVCenter">
    <widget class="QPushButton" name="btnSave">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string>primary</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblErrorMsg">
     <property name="text">
      <string>Fields cannot be left blank. </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="txtStartupAppComment">
     <property name="placeholderText">
      <string>App Comment</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="txtStartupAppName">
     <property name="placeholderText">
      <string>App Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="txtStartupAppCommand">
     <property name="placeholderText">
      <string>Command</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblTitle">
     <property name="accessibleName">
      <string>dialog-title</string>
     </property>
     <property name="text">
      <string>Application</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtStartupAppName</tabstop>
  <tabstop>txtStartupAppComment</tabstop>
  <tabstop>txtStartupAppCommand</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
